src/pivot-table/components/cells/utils/get-adjuster-color.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-unsafe-enum-comparison */
+import { COLORING } from "@qlik/nebula-table-utils/lib/utils";
+import { color } from "d3-color";
+import type { StyleService } from "../../../../types/types";
+import { Colors } from "../../shared-styles";
+
+/**
+ * Converts border color to opaque color,
+ * by alpha blending the border color with the background the header or top grid background.
+ * The background is assumed to be opaque.
+ * If the background is transparent (which is the default color) it is assumed to be white
+ */
+export default function getAdjusterColor(styleService: StyleService, isHeader: boolean) {
+  const background = isHeader ? styleService.header.background : styleService.dimensionValues.background;
+  const opaqueBackground = background === Colors.Transparent ? COLORING.WHITE : background;
+  const borderColor = color(styleService.grid.border)?.rgb();
+  const backgroundColor = color(opaqueBackground)?.rgb();
+
+  if (backgroundColor && borderColor && borderColor.opacity < 1) {
```

Comment:
Think you can do

```suggestion
  if (backgroundColor && borderColor?.opacity < 1) {
```

Reply:
I think I got a lint error, will check

Reply:
then borderColor can be undefined, we don't want that

Reply:
But if `borderColor` is undefined in `borderColor?.opacity < 1`. It should always faild the less then check?